src/js/media-chrome-range.js

```diff
@@ -169,7 +168,7 @@ class MediaChromeRange extends MediaChromeHTMLElement {
   */
   getBarColors() {
     const range = this.range;
-    const rangePercent = (this.range.value / 1000) * 100;
+    const rangePercent = (range.value / range.max) * 100;
```

Comment:
this was an intentional change (forward looking for aria stuff and values more directly corresponding with things like time/volume) but could be thought of as out of scope. Let me know if you want me to back it out @heff

Reply:
Doesn't seem like it actually changes any functionality, so seems fine. 🤷 I have found that keyboard control sucks when the increment is 1/1000. But we want it to be smoother than 1/100 otherwise.